--- hyprwayland-scanner-0.4.0/CMakeLists.txt.orig	2024-07-05 23:44:17.000000000 +0200
+++ hyprwayland-scanner-0.4.0/CMakeLists.txt	2024-07-28 19:44:00.204956759 +0200
@@ -18,7 +18,6 @@
   message(STATUS "Configuring hyprwayland-scanner in Debug with CMake")
   add_compile_definitions(HYPRLAND_DEBUG)
 else()
-  add_compile_options(-O3)
   message(STATUS "Configuring hyprwayland-scanner in Release with CMake")
 endif()
 
